pyo3-macros-backend/src/attributes.rs

```diff
@@ -154,8 +154,8 @@ pub fn get_pyo3_options<T: Parse>(attr: &syn::Attribute) -> Result<Option<Punctu
 }
 
 pub fn is_attribute_ident(attr: &syn::Attribute, name: &str) -> bool {
```

Comment:
Looking at this, I think this function can just be removed at call sites and replaced with `attr.path().is_ident(name)`?